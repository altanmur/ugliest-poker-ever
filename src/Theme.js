import React from 'react';
import PropTypes from 'prop-types';
import { ThemeProvider } from 'styled-components';

export const theme = {
  colors: {
    indigo: '#4A0968',
    blackcurrant: '#27172F',
  },
};

const Theme = ({ children }) => (
  <ThemeProvider theme={theme}>{children}</ThemeProvider>
);

Theme.propTypes = {
  children: PropTypes.element.isRequired,
};

export default Theme;
