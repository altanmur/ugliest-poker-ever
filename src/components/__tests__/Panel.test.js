import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Panel from '../Panel';
import { theme } from '../../Theme';

describe('Panel Component', () => {
  it('should render correctly', () => {
    const component = shallow(<Panel theme={theme} />);

    expect(component).toMatchSnapshot();
  });
});
