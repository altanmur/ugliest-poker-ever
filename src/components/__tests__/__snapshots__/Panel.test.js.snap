// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel Component should render correctly 1`] = `
<styled.section>
  <styled.section />
  <styled.img
    src="/img/coins.png"
  />
</styled.section>
`;
